couchexport/util.py

```diff
@@ -45,6 +45,10 @@ def clear_attachments(schema_or_doc):
         del schema_or_doc['_attachments']
     if schema_or_doc and 'case_attachments' in schema_or_doc:
         del schema_or_doc['case_attachments']
+    if schema_or_doc:
```

Comment:
@czue @dannyroberts I updated below, but after updating I don't think a question named attachments would get removed.  That questions would either be under `updated_known_properties` or `updated_unknown_properties`, correct?